install/linux/vendor-pymupdf.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+import sys
+from pathlib import Path
+
+DZ_VENDOR_DIR = Path("./dangerzone/vendor")
+
+
+def main():
+    parser = argparse.ArgumentParser()
```

Comment:
I've seen in other parts of the code that we're using click, and I'm wondering why it's not the case here? 🤔

Reply:
Where I can, I try not to introduce external libraries for our dev scripts, so that we don't need enter a Poetry environment just to run them.

Reply:
Thanks for clarifying this. I would like to challenge this, if that's open for discussion?

I believe using the same tooling across the project would be beneficial in terms of general maintenance, as it would require from the dev team to only know one tool, rather than two, dig deeper in their docs etc.

That being said, that's the current status of the code, and I don't think it's worth blocking the work here, so let's discuss that the next time we meet!

Reply:
Sure, let's discuss when we meet. I get what you're saying, but I always kinda believed that `argparse`, being in stdlib, is ubiquitous. So, it feels to me more like the default option, instead of "a different option". Also, given that this is a dev script, I figured it's best not to complicate things more. We now have lots of dev scripts though, so let's see what makes sense in the long run.

Reply:
I agree about `argparse` being seen as the default option. It's part of the batteries included with python. I've been using it extensively in the past, and I mostly like it. I also can see why and how `click` enhances this situation (and I've also used it on different projects).

To me the question seems to be mainly about which one we decide on using, and sticking with one rather than having two options depending on the code base. My preference is to use tools like `click` and `requests` because they solve real problems of their stdlib counterparts.

I would like to know more about why you prefer not relying on dependencies for the dev scripts. (I might be missing an obvious reason why)

Reply:
Closing this to not block the merge. Discussion is happening here: https://github.com/freedomofpress/dangerzone/issues/946